src/ucs/nvtx/Makefile.am

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright (C) Mellanox Technologies Ltd. 2020.  ALL RIGHTS RESERVED.
+# Copyright (c) 2021, NVIDIA CORPORATION. All rights reserved.
+#
+# See file LICENSE for terms.
+#
+
+if HAVE_CUDA
+
+module_LTLIBRARIES      = libucs_nvtx.la
+libucs_nvtx_la_CPPFLAGS = $(BASE_CPPFLAGS) $(CUDA_CPPFLAGS)
+libucs_nvtx_la_CFLAGS   = $(BASE_CFLAGS) $(CUDA_CFLAGS)
+libucs_nvtx_la_LIBADD   = $(CUDA_LIBS) \
```

Comment:
do we really need to link with cuda?

Reply:
Not needed but included these because I had added -lnvToolsExt into CUDA_LIBS. I'll change this.